DrinkManagerWeb/Views/Drinks/SearchByIngredients.cshtml

```diff
@@ -0,0 +1,23 @@
+﻿@model DrinkManagerWeb.Models.ViewModels.DrinksViewModel
+
+@{
+    ViewBag.Title = "Search by ingredients";
+    Layout = "_Layout";
+}
+
+<div>
+    <form asp-controller="Drinks" asp-action="SearchByIngredients" method="get">
+        <p>
+            Enter ingredients to find <br/>(Tip 1: you can provide one or more ingredients separated by a space) <br/>
+            (Tip 2: choose if you want to find drinks containing ANY or ALL provided ingredients):<br/>
+            <input type="text" name="SearchString">
+            <input type="radio" id="any" name="searchCondition" value="any" checked="checked" />
+            <label for="all">Any ingredient</label>
+            <input type="radio" id="all" name="searchCondition" value="all" />
+            <label for="any">All ingredients</label>
```

Comment:
I think labels are mixed - any for all and all for any, instead of any for any and all for all

Reply:
Fixed.